### PR TITLE
reorder python builds

### PR DESCRIPTION
**Pull request by michalskrivanek**

### CODE REVIEW

**Comment by michalskrivanek** [comment]
https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/build/5601975/

**Comment by michalskrivanek** [comment]
eh, https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/build/5601978/

File: ovirt-imageio.spec.in [hunk 1]
```diff
@@ -41,40 +41,44 @@ Transfer disk images on oVirt system.
...
 %if %{with_python311}
 %define python3_pkgversion 3.11
 %define __python3 /usr/bin/python3.11
 %py3_install
-%define python3_pkgversion 3
-%define __python3 /usr/bin/python3
 %endif
 
-%py3_install
+define python3_pkgversion 3
```
Comment (aesteve-rh) [inline ovirt-imageio.spec.in]:
```suggestion
%define python3_pkgversion 3
```

**Comment by michalskrivanek** [comment]
seems it works. the requirements are now correct for both 3.9 and 3.11 on el9

**Comment by michalskrivanek** [comment]
@aesteve-rh please re-review

**Comment by aesteve-rh** [comment]
Looks good to me, thanks @michalskrivanek !
Could you please squash and sign the commit to make the pipelines to pass?

**Comment by michalskrivanek** [comment]
nah, I can only click around in GUI and there's no way how to do that;-)

**Comment by nirs** [comment]
Replaced by #187, lets continue there.

**Pull request closed without merging**